### PR TITLE
Align asset and structure properties

### PR DESCRIPTION
**Pull request by kmp091**
We're treating assets and structures as one "type" in the application context. It'll be easier at this time to align their properties with one another.

This means removing a few properties that we aren't using and adding others that are being used.
**Asset**
- removed Network ID

**Structure**
- removed Land ID
- added Linear Reference Point and Line Locations

### CODE REVIEW

File: Ontology/Willow/Asset/Asset.json [hunk 1]
```diff
@@ -282,33 +270,6 @@
       "schema": "date",
       "writable": true
     },
-    {
```
Comment (rszcodronski) [inline Ontology/Willow/Asset/Asset.json]: We need to keep IP Address, MAC Address, and modelNumber. Even if not every Asset twin defines these, they are fundamental to a basic Asset definition.

File: Ontology/Willow/Asset/Asset.json [hunk 2]
```diff
@@ -318,24 +279,6 @@
       "schema": "string",
       "writable": true
     },
-    {
-      "@type": "Property",
```
Comment (rszcodronski) [inline Ontology/Willow/Asset/Asset.json]: We also need to keep serialNumber and software as base properties.

**Review by rszcodronski — COMMENTED** [review]
Overall looks good. Agree with removing the landID, networkID. Also agree with the properties added to Structure. I added a few comments on properties we need to keep in the base Asset model.

File: Ontology/Willow/Asset/Asset.json (hunk 1 above)
Comment (kmp091) [reply Ontology/Willow/Asset/Asset.json]: These properties have been restored.

File: Ontology/Willow/Asset/Asset.json (hunk 2 above)
Comment (kmp091) [reply Ontology/Willow/Asset/Asset.json]: These properties have been restored.

**Pull request merged**